test/test_game.py

```diff
@@ -280,9 +290,96 @@ def test_penalization_after_invalid_moves(
         self.assertEqual(game.game_is_active, game_active)
 
     def test_board_game_is_class(self):
-        game = WumpusGame()
+        game = patched_game()
         self.assertIsInstance(game._board, Board)
 
+    @parameterized.expand([
+        ('shoot opponent', 5, 8, 8, WEST, 8, 7, 16100, 4, "     "),
+        ('shoot empty cell', 5, 8, 8, EAST, 8, 9, 1100, 4, "##F##"),
+        ('shoot hole', 5, 8, 8, NORTH, 7, 8, 1100, 4, "  O  "),
+        ('shoot own char', 5, 8, 8, SOUTH, 9, 8, 900, 4, "  B  "),
+    ])
+    def test_execute_action_shoot(self, name,
+                                  initial_arrows,
+                                  from_row, from_col, direction,
+                                  destiny_row, destiny_col,
+                                  expected_score, expected_arrows, expected_destiny_cell):
```

Comment:
`Destiny` is "the hidden power believed to control future events". You may be referring to `Destination` "the place to which someone or something is going or being sent"
This happens in multiple tests. I'm only leaving a comment here but review all tests

Reply:
Exellent, I'll change it